$MirOS: ports/archivers/xz/patches/patch-src_xz_message_c,v 1.1 2010/12/11 20:33:27 tg Exp $
--- src/xz/message.c.orig	Fri Jun 22 06:22:06 2012
+++ src/xz/message.c	Thu Oct 18 15:24:56 2012
@@ -1082,7 +1082,7 @@ message_version(void)
 	// line tool version, so print both.
 	if (opt_robot) {
 		printf("XZ_VERSION=%" PRIu32 "\nLIBLZMA_VERSION=%" PRIu32 "\n",
-				LZMA_VERSION, lzma_version_number());
+		    (uint32_t)(LZMA_VERSION), lzma_version_number());
 	} else {
 		printf("xz (" PACKAGE_NAME ") " LZMA_VERSION_STRING "\n");
 		printf("liblzma %s\n", lzma_version_string());
